pkg/plugin/manifest/manifest.go

```diff
@@ -44,7 +44,7 @@ func (s *SonobuoyConfig) DeepCopy() *SonobuoyConfig {
 type Manifest struct {
 	SonobuoyConfig SonobuoyConfig `json:"sonobuoy-config"`
 	Spec           Container      `json:"spec"`
-	ExtraVolumes   []Volume       `json:"extra-volumes"`
+	ExtraVolumes   []Volume       `json:"extra-volumes,omitempty"`
```

Comment:
This was because otherwise, we got a slightly ugly (and unnecessary) `extra-volumes: []` which we didn't normally see in our manually written YAML.